.gitignore

```diff
@@ -21,3 +21,5 @@ ketcher
 protobuf-*
 node_modules
 package-lock.json
+# docker-compose environment variables
+.env
```

Comment:
I have never told git to ignore a file I had also committed. I wonder what will happen.

Reply:
It should require `git add -f` in new branches I hope...